tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -971,6 +971,7 @@ cc_library(
         "//tensorflow/compiler/xla:xla_data_proto_cc",
         "//tensorflow/compiler/xla/hlo/ir:hlo",
         "//tensorflow/tsl/platform:statusor",
+        "//tensorflow/tsl/platform:tensor_float_32_hdr_lib",
```

Comment:
Adding this dependency caused a linker error. I have not yet figured out how to fix it.

Reply:
So to fix the linker error, I think we need to add:

```
+ if_static(["//tensorflow/tsl/platform:tensor_float_32_utils"]),
```

Because this is the target that contains the definition of the tensor_float_32_execution_enabled() function.
However adding this dependency will enable tensor_float_32 by default for anything that links in this matmul_util target. Is this intended?

Reply:
Yes, it is intended that tensor_float_32 be enabled by default.